.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,4 +15,10 @@ greatly reduce the amount of work needed to review your work. -->
 1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
 1. [ ] The changes are tested.
 1. [ ] The code is documented (docstrings, project documentation).
-1. [ ] Changes are mentioned in the changelog.
\ No newline at end of file
+1. [ ] Changes are mentioned in the changelog.
+
+### Test environment config
+
+```
+API_URI=https://master.staging.saleor.rocks/graphql/
```

Comment:
Having it as text than code would be nicer (refer to the suggestion @ https://github.com/mirumee/saleor-storefront/pull/800/files#r450766781), we could click the link rather than copy/paste.